ext/phar/tests/gh16406.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
+GH-16406 (Assertion failure in ext/phar/phar.c:2808)
+--EXTENSIONS--
+phar
+zlib
+--INI--
+phar.readonly=0
+phar.require_hash=0
+--FILE--
+<?php
+$fname = __DIR__ . '/gh16406.phar';
+@unlink($fname . '.tar');
+@unlink($fname . '.gz');
+@unlink($fname);
```

Comment:
These shouldn't be required, but won't hurt.

Reply:
So the reason I added this is because when I was writing the test I first had messed up the clean section, which meant the test wouldn't execute anymore.